citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -13,23 +13,26 @@
 	    visiblePages: 10,
 	    initiateStartPageClick: false,
 	    onPageClick: function (event, page) {
-	    	window.location.href = "items?page=" + page;
+	        var currentURL = window.location.href;
+	        if (currentURL.match(/search/)) {
+	            var key = $("#search").val();
+	            if (key != undefined && key != '') {
+	                window.location.href = "search?searchTerm=" + key + "&page=" + page;
+	            }
+	        } else {
+	            var queryParams = window.location.search;
+	            if(!queryParams) {
+	                queryParams = "?";
+	            } else {
+	                const params = new URLSearchParams(queryParams);
+	                params.delete('page');
+	                queryParams = "?" + params.toString();
+	            }
```

Comment:
the above code should come first, then the check for the url (so all query parameters are always passed on)

Reply:
I'm making this change. But now that I'm testing this, I find many places where these queryparams get lost. For e.g., when navigating between next and prev citations, or going back from item to items list, etc.